src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -189,5 +190,54 @@ private void HandleMessage(Message message, IRequest request, IIssueConsumer con
             // rules will never be active in a quality profile.
             return rulesConfiguration.ActivePartialRuleKeys.Contains(message.RuleKey, CFamilyShared.RuleKeyComparer);
         }
+
+        internal /* for testing */ static void ExecuteSubProcess(Action<Message> handleMessage, IRequest request, IProcessRunner runner, ILogger logger, CancellationToken cancellationToken)
```

Comment:
Moved from `CFamilyHelper`.

No changes, other than renaming the method and using `SubProcessFilePaths` to get the file paths.